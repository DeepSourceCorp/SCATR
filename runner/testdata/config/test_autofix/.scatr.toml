test_autofix = false

[autofix]
script = "script"
