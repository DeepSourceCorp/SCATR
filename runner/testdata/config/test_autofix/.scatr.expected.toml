test_autofix = false

[checks]
interpreter = "sh"

[autofix]
script = "script"
interpreter = "sh"

[processor]
interpreter = "sh"
